File: 251127/연속되는 수 4/continuous-number4.py
n = int(input())
arr = [int(input()) for _ in range(n)]

count, result = 0, 0
for i in range(n):
    if i >= 1 and arr[i] > arr[i - 1]:
        count += 1
    else:
        count = 1
    result = max(count, result)

print(result)

File: 260320/연속되는 수 4/continuous-number4.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');

const n = Number(input[0]);
const arr = input.slice(1, 1 + n).map(Number);

let count = 0, answer = 0;
for (let i = 0; i < arr.length; i++) {
    if (i >= 1 && arr[i - 1] < arr[i]) {
        count += 1; 
    } else {
        count = 1;
    }

    answer = Math.max(count, answer);
}

console.log(answer);